tests/integration/features/istio/evaluation/istio_installation_demo.feature

```diff
@@ -9,13 +9,13 @@ Feature: Installing Istio module with evaluation profile
     When Istio CR "istio-sample" from "istio_cr_template" is applied in namespace "kyma-system"
     Then Istio CR "istio-sample" in namespace "kyma-system" has status "Ready"
     And Istio CR "istio-sample" in namespace "kyma-system" has condition with reason "ReconcileSucceeded" of type "Ready" and status "True"
-    Then "proxy" has "requests" set to cpu - "10m" and memory - "32Mi"
-    And "proxy" has "limits" set to cpu - "250m" and memory - "254Mi"
-    And "ingress-gateway" has "requests" set to cpu - "10m" and memory - "32Mi"
+    #Then "proxy" has "requests" set to cpu - "10m" and memory - "32Mi"
```

Comment:
Int tests for egress and proxy - follow up

Reply:
No, this should stay, as the tests should be implemented in a follow up.
This feature should not change, only the implementation